src/components/form/Volunteer.jsx

```diff
@@ -11,9 +11,19 @@ import { STATUSES } from "@/data/admin/Volunteers.js";
 const Volunteer = () => {
   const { data: session } = useSession();
 
+  function extractFirstName(str) {
+    return str.trim().split(" ")[0];
+  }
+
+  function extractLastName(str) {
+    return str.trim().split(" ")[1];
+  }
+
   const [volunteer, setVolunteer] = useState({
```

Comment:
for each form, when we submit, we need to remove the first and last separate and merge it back into just name so the DB remains the same